src/ducks/apps/components/ApplicationPage/index.jsx

```diff
@@ -25,7 +26,7 @@ import withBreakpoints from 'cozy-ui/transpiled/react/helpers/withBreakpoints'
 const MOBILE_PLATFORMS = ['ios', 'android']
 const isMobilePlatform = name => MOBILE_PLATFORMS.includes(name)
 
-const { BarCenter } = cozy.bar
+const { BarCenter } = cozy.bar || {}
```

Comment:
same here, you should add a comment / context to explain that. We'll not remember in a few weeks

Reply:
J'ajoute un commentaire 👍